### DISCUSSION TITLE
Map rotation at standstill in navigation mode

### DISCUSSION THREAD

**Discussion by GE-SB59** [osmandapp/OsmAnd#17418] [Q&A]
When navigating using map rotation in moving direction at standstill the map starts rotating irregular.
It seems the GPS direction signal changes even at standstill in my Android Moto. 
Is there any method to avoid this map turning when the speed is less than a specified value.

I'm German so please accept my simple english.

**Comment by mikehgentry** [comment]
There are some suggestions in #17053

**Comment by GE-SB59** [comment]
@mikehgentry 
Thank you for the hint.
The last comment of @mdykierek
_.... It seems that from time to time, for a brief moment the speed displays 1 km/h (yet the position doesn't change ;)) and the dot changes to arrow._
describes the issue exactly. When the arrow appears, the map starts to rotate. The speed displays values of up to 10 km/h without changing position.
For me this effect, wich happens nearly at every traffic light, is very annoying and makes the use of OSMAND as car navigation tool more or less useless. My TomTom does not show this effect, so it seems there are feasable solutions.
The tracking tool uses several threshold values to generate smooth tracks. Why not using such thresholds, especially a distance threshold, to decide to stop rotating the map.
